GroupBundle/Document/Group.php

```diff
@@ -78,10 +79,10 @@ public function __clone()
     {
         $this->id = null;
         $this->initCollections();
-        $this->setName($this->cloneLabel($this->name));
+        $this->setName($this->name . '_' . uniqid());
```

Comment:
Because, if you clone twice from the same group, it will generate two "label_2" which will result in a unique index exception from fos.